event_attribution_reporting_views.md

```diff
@@ -43,10 +43,11 @@ The browser will expose a new interface:
 
 ```
 dictionary AttributionSourceParams {
-  required DOMString attributionSourceEventId;
   required USVString attributionDestination;
-  optional USVString attributionReportTo;
-  optional unsigned long attributionExpiry;
+  required DOMString attributionSourceEventId;
+  USVString attributionReportTo;
+  unsigned long long attributionExpiry;
```

Comment:
This should just be a long long per https://wicg.github.io/conversion-measurement-api/#monkeypatch-anchor

Reply:
Done. Also replaced `USVString` with `DOMString` and reordered for consistency with that link.